checkov/terraform/checks/resource/aws/KeyspacesTableUsesCMK.py

```diff
@@ -14,7 +18,7 @@ def scan_resource_conf(self, conf) -> CheckResult:
         if conf.get("encryption_specification") and isinstance(conf.get("encryption_specification"), list):
             encrypt = conf.get("encryption_specification")[0]
             if encrypt.get("kms_key_identifier") and isinstance(encrypt.get("kms_key_identifier"), list):
-                if encrypt.get("type") == ["CUSTOMER_MANAGED_KEY"]:
+                if encrypt.get("type") == ["CUSTOMER_MANAGED_KMS_KEY"]:
```

Comment:
```suggestion
                if encrypt.get("type") == ["CUSTOMER_MANAGED_KMS_KEY"] or encrypt.get("type") == ["CUSTOMER_MANAGED_KEY"]:
```

@tal66 So what I meant is this

Reply:
but the aws doc linked in the issue says it's not valid, why keep both?

Reply:
chipping in 🙂 it is `CUSTOMER_MANAGED_KMS_KEY` indeed `CUSTOMER_MANAGED_KEY` doesn't exist as can seen in the official API docs https://docs.aws.amazon.com/keyspaces/latest/APIReference/API_CreateTable.html#keyspaces-CreateTable-request-encryptionSpecification

Reply:
My bad, I see it now. Thanks!